src/lib/adopted-probes.ts

```diff
@@ -179,6 +184,7 @@ export class AdoptedProbes {
 		const isCustomCity = adoptedProbe.isCustomCity;
 
 		if (!connectedProbe) {
+			await this.updateProbeData(adoptedProbe, { status: 'offline' });
```

Comment:
This seems like it will run an update every time we call this function even if it's already set to `offline`? 

Also, other status changes are already propagated correctly (e.g. QA fail)?

Reply:
Fixed regular "offline" updates. Other statuses are propagated correctly.